app/models/pillar.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+# Pillar represents a pillar value on Salt.
```

Comment:
I think it would be good to add an example and the actual use case here. I would like to understand why this is needed and what kind of pillar data we push this way.